### ISSUE TITLE
Language cutoff unmet despite (language) profile settings

### ISSUE THREAD

**Issue by Cheezzhead** [Sonarr/Sonarr#4840]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

**Bug Description**
I'm working through my long list of cutoff unmet episodes, and I'm having some problems with marking international (non-english) shows as cutoff met. Either there's a bug or I'm misunderstanding the usage of language profiles.

**Screenshots**
![LP-International](https://user-images.githubusercontent.com/2292563/148379361-32df79c6-b2d7-41ef-9f5e-76c76fbac37f.png)
I have a catch-all profile for non-english shows named 'International'.

![AiB-Edit](https://user-images.githubusercontent.com/2292563/148379353-4e32afc8-a420-42c3-9880-65743f458057.png)
An example show with the language profile set to 'International'.

![AiB-Manage](https://user-images.githubusercontent.com/2292563/148379330-cdd716ce-8ac3-4b64-836b-5811e18ff551.png)
Episodes in japanese language are marked as cutoff unmet.

### Expected Behavior

Episodes with a languages that's checked in the language profile should be marked as having its cutoff met, I think? At least when the 'Upgrades Allowed' box is unchecked.

If this is not the expected behavior, how would I go about marking these episodes? I would prefer to avoid having to create a separate language profile for each language.

### Environment

```markdown
- OS: Ubuntu 20.04
- Sonarr: 3.0.6.1342  [package version ls124 by linuxserver.io]
- Docker Install: Yes
- Using Reverse Proxy: Yes
- Browser: Firefox 95.0.2
```

### What branch are you running?

Main

### Trace Logs?

Changing language profile does not produce any trace log entries

### Anything else?

Changing language profile does not produce any trace log entries

**Comment by bakerboy448** [comment]
The Cutoff unmet filter is for quality - not language.

**Comment by Cheezzhead** [comment]
<img width="1661" alt="Schermafbeelding 2022-01-06 om 15 08 16" src="https://user-images.githubusercontent.com/2292563/148395490-b7c32a44-eab5-4d56-98c8-605d957ae49a.png">

It's showing up in my cutoff unmet filter despite having its quality cutoff met, though?

**Comment by Taloth** [comment]
> The Cutoff unmet filter is for quality - not language.

False, it's for both.

**Comment by Taloth** [comment]
@Cheezzhead Can you check which language is the cutoff when you enable Upgrades Allowed? If it's not English, then change it to English and disable Upgrades Allowed again.

**Comment by Cheezzhead** [comment]
This seems to have done the trick, thanks!

Proper workaround until a fix is pushed, I couldn't ask for more haha. Thanks for the help guys! If you need any other info let me know.

**Comment by markus101** [comment]
This was fixed in 97e40dc00ac5f284c9765c6a93a23caba60f149d
